docs-parts/admin/5-blob-config_lang3.rst

```diff
@@ -1,3 +1 @@
-.. code-block:: python
-
-    >>> schema.external_table.clean_store('external-name')
+   This is done using the ``set_cache_folder`` method of the schema object.
```

Comment:
What `set_cache_folder` method?

Reply:
No idea. I only moved this line from the general content into a language-specific file.

Reply:
There does not seem to be such a method. Do you know what the method should be?

Reply:
```suggestion
This is done by saving the path in the ``"cache"`` key of the DataJoint configuration dictionary:

.. code-block:: python

   dj.config['cache'] = '/temp/dj-cache
```